src/test/jmh/io/lettuce/core/protocol/CommandHandlerBenchmark.java

```diff
@@ -46,81 +47,91 @@
     private final static ClientOptions CLIENT_OPTIONS = ClientOptions.create();
     private final static EmptyContext CHANNEL_HANDLER_CONTEXT = new EmptyContext();
     private final static byte[] KEY = "key".getBytes();
+    private static final String VALUE = "value\r\n";
     private final EmptyPromise PROMISE = new EmptyPromise();
 
     private CommandHandler commandHandler;
-    private Command command;
```

Comment:
Commands were pre-created to not include fixture setup in the actual benchmark. We should keep the buffers pre-allocated and only reset the reader index to exclude the cost of buffer allocations and such from the actual benchmark.

Reply:
But `CommandHandler` tries to release the buffer after reading - I can manually increase the `refCnt` by one after each run and then manually release the buffers in `tearDown` - I just thought it's a little bit ugly:) But I can do it if you'd like?
Also, is there a way to reset `command` state so that it can also be reused? (After one write it becomes not writable and `commandHandler` skips them, that's why I create them from scratch on every run)